### PR TITLE
Improve Distribution

### PR DESCRIPTION
**Pull request by FiberJW**
Hey Jared!

I suggest that we use NPM's built-in hooks and binary distribution instead of having users clone and build a binary on their own. This makes it even easier for users to get started! They can install it on a global or project-scoped basis this way and also run it on their own project's hooks (npm run build, etc.) too 👌 

Thanks for checking this PR out,

Juwan

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -17,74 +16,65 @@ Usage:
...
 
-- [ ] Interactive code samples!
-  - [ ] First pass - include the compiled javascript for code samples so you can see the result
-  - [ ] Second pass - make the code samples editable, with an in-page compiler so you can experiment
-- [ ] Parse the bsconfig.json and package.json of a project
-  - [x] to determine the github url
-    - [ ] support all types of github specifications https://docs.npmjs.com/files/package.json
-  - [ ] name, source directories, etc.
-- [ ] Handle ocamldoc (see included octavius source) so we can build ocamly projects too
+* [ ] Interactive code samples!
```
Comment (jaredly) [inline Readme.md]: why this change?

**Comment by jaredly** [comment]
Yeah, this is a good idea. I'm going to take another approach because I've seen npm/yarn be somewhat flaky with postinstall binary symlinking. But thanks for the nudge!

File: Readme.md (hunk 1 above)
Comment (fiber-god) [reply Readme.md]: ah that's what happened when it ran through Prettier. It has no effect, but I'll revert it if you want 👍🏾
